.github/workflows/publish.yml

```diff
@@ -25,10 +25,10 @@ jobs:
           run_install: false
       - name: Build Monorepo and Publish
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.MIA_NPM_REGISTRY_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: |
           pnpm i --frozen-lockfile
           pnpm unit
           pnpm build
-          pnpm -r publish --access restricted --registry ${{ secrets.MIA_NPM_REGISTRY_URL }} --no-git-checks
+          pnpm -r publish --access public --no-git-checks
```

Comment:
This change is for this: https://docs.npmjs.com/cli/v10/commands/npm-access#details